src/features/auth/signup/NextBeneficiaryStep/NextBeneficiaryStep.tsx

```diff
@@ -5,12 +5,10 @@ import { useBeneficiaryValidationNavigation } from 'features/auth/signup/useBene
 import { LoadingPage } from 'ui/components/LoadingPage'
 
 export function NextBeneficiaryStep() {
-  const { error, navigateToNextBeneficiaryValidationStep } = useBeneficiaryValidationNavigation()
-  useFocusEffect(navigateToNextBeneficiaryValidationStep)
-
-  if (error) {
-    throw error
-  }
+  const { navigateToNextBeneficiaryValidationStep } = useBeneficiaryValidationNavigation()
+  useFocusEffect(() => {
+    navigateToNextBeneficiaryValidationStep
```

Comment:
```suggestion
    navigateToNextBeneficiaryValidationStep()
```

sinon ça va pas appeler la fonction. Mais le meilleure moyen de savoir c'est de tester

Reply:
ou tu peux laisser `useFocusEffect(navigateToNextBeneficiaryValidationStep)` sinon

Reply:
oupssss bien vu